patches/api/0334-Removed-ItemStackRecipeChoice.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Auxilor <william.favierparsons1@gmail.com>
+Date: Sun, 29 Aug 2021 21:16:51 +0100
+Subject: [PATCH] Removed ItemStackRecipeChoice
```

Comment:
Remove the [original patch](https://github.com/PaperMC/Paper/blob/master/patches/api/0153-Add-ItemStackRecipeChoice-Draft-API.patch) instead.

Reply:
on it